hw_6_rgb_diod_potentiometr/src/main.cpp

```diff
@@ -0,0 +1,69 @@
+#include <Arduino.h>
+#include <NanitLib.h>
+
+#define red P4_2
+#define green P4_4
+#define blue P4_3
+#define potPin P4_1
+
+
+
+void setup() {
+  Nanit_Base_Start();
+  Serial.begin(9600);
+  pinMode(red, OUTPUT);
+  pinMode(green, OUTPUT);
+  pinMode(blue, OUTPUT);
+  pinMode(blue, OUTPUT);
+}
+
+void set_rgb(byte r, byte g, byte b);
+
+void loop() {
+  int pot = analogRead(potPin);
+  int pot_map = map(pot, 0, 1023, 0, 1022);
+  int base_devider = 146;
```

Comment:
некритично але апічятка

```suggestion
  int pot_map = map(pot, 0, 1023, 0, 1022);
  int base_divider = 146;
```